### DISCUSSION TITLE
Elementary OS is using the wrong color profile on my monitor

### DISCUSSION THREAD

**Discussion by Mashrafi2006** [elementary/triage#210] [Q&A]
I recently installed Elementary OS and noticed that the colors on my monitor don't look right. It seems like the operating system is using the wrong color profile. I'm not sure how to change the color profile to make it look correct.

Can someone help me with this issue? How can I change the color profile on Elementary OS to make it use the correct one for my monitor? Thanks in advance for your help.

**Comment by migtorr** [comment]
https://reddit.leptons.xyz/r/elementaryos/comments/11osvpy/blue_tint_on_fresh_install/jbx69ax/?context=3

**Reply by MH0386** [reply]
Don't work for me
